ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -211,7 +211,7 @@ def _wan_policy_key(self) -> str:
         """
         return "policy" if self.shared_utils.wan_mode == "cv-pathfinder" else "path_selection_policy"
 
-    def _generate_wan_load_balance_policy(self, name: str, input_dict: dict, context_path: str) -> dict:
+    def _generate_wan_load_balance_policy(self, name: str, input_dict: dict, context_path: str, ignore_priority_1: bool = False) -> dict:
```

Comment:
@ayushmittal-arista I had to add this to rebase for now but we need to figure this out